src/TwigComponent/doc/index.rst

```diff
@@ -1046,6 +1046,79 @@ And in your component template you can access your embedded block
         {% block footer %}{% endblock %}
      </div>
 
+Anonymous TwigComponent
+-----------------------
+
+Sometimes, reusable elements do not require complex logic or injected service to render what it could be processed
+from one template itself. (e.g. retrieving a customized primary button that could take a different label).
+
+No need for class. Your component matches the namespace of the file you created where components live (see `Twig Template Namespaces`_).
+
+.. code-block:: html+twig
+
+    {# templates/components/Button/Primary.html.twig #}
+    <button class="primary">
+        {% block content %}
+        {% endblock %}
+    </button>
+
+Then use your component with ``:`` to navigate through the directories:
+
+.. code-block:: html+twig
+
+     {# index.html.twig #}
+     ...
+    <div>
+       <twig:Button:Primary>
+           Click Me!
+       </twig:primary>
+    </div>
+
+Now let's pass props to your button by defining a ``{% props %}`` tag in its view.
```

Comment:
I think we should first show passing a normal "attribute" and show how that works very normally (show the `{{ attributes }}` being printed in the template). Maybe a `role` attribute, for example.

Then we can show how you can, in addition to an attribute, pass a prop using `{% props}`.